cypress/e2e/CMS/$BlogSeries/index.js

```diff
@@ -111,6 +124,30 @@ And('blog series description reads {string}', (desc) => {
     cy.get(`p:contains('${desc}')`).should('be.visible');
 });
 
+And('user clicks on {string} sub tab', (contentSubTab) => {
+    cy.get(`ul.admin-list li a:contains("${contentSubTab}")`).click({ force: true });
+});
+
+And('user selects {string} option from Operations for {string}', (option, label) => {
+    cy.get('[data-drupal-selector="edit-vocabularies-cgov-blog-topics"] > :nth-child(4) > .dropbutton-wrapper > .dropbutton-widget > .dropbutton > .list > a').click({ force: true });
+});
+
+And('user selects {string} radio button under {string}', (option, label) => {
+    cy.get(`#edit-field-owner-blog label:contains("${option}")`).first().click();
+});
+
+And('user checks {string} checkbox', (checkboxLbl) => {
+    cy.get('fieldset#edit-field-blog-topics--wrapper input').parent().find(`label:contains("${checkboxLbl}")`).as('checkbox');
+    cy.get('@checkbox').parent().find('input').check();
+});
+
+And('user clicks on the tool bar status red button {string}', (status) => {
+    cy.get(`a[data-label='${status}']`).click();
```

Comment:
so this has the same code implementation as 2 more steps - I suggest re-using step in common 
user clicks on the tool bar status green button {string} - the color doesnt matter, the status is! and it is a variable, so we are good 

also, remove existing steps that say red and yellow button respectfully and update them to be green instead

Reply:
Fixed by updating the feature file and removing the code from index.js